shared-bindings/displayio/Bitmap.c

```diff
@@ -62,14 +62,11 @@
 //|         ...
 STATIC mp_obj_t displayio_bitmap_make_new(const mp_obj_type_t *type, size_t n_args, size_t n_kw, const mp_obj_t *all_args) {
     mp_arg_check_num(n_args, n_kw, 3, 3, false);
-    uint32_t width = mp_obj_get_int(all_args[0]);
-    uint32_t height = mp_obj_get_int(all_args[1]);
-    uint32_t value_count = mp_obj_get_int(all_args[2]);
+    uint32_t width = mp_arg_validate_int_range(mp_obj_get_int(all_args[0]), 1, 32767, MP_QSTR_width);
+    uint32_t height = mp_arg_validate_int_range(mp_obj_get_int(all_args[1]), 1, 32767, MP_QSTR_height);
+    uint32_t value_count = mp_arg_validate_int_range(mp_obj_get_int(all_args[2]), 1, 32767, MP_QSTR_value_count);
```

Comment:
I am second guessing this maximum on the `value_count` Prior to this change it was possible to pass a number up to `65536`. The docs make mention of 8, 16, and 32 bits per pixel in the bitmap and capping this at 32767 makes it so it wouldn't be possible (or at least easy to create) a Bitmap that has 32bits per pixel unless I'm misunderstanding how something works. 

I'm still trying to wrap my head around how Bitmap + ColorConverter function together as compared to Bitmap + Palette which may factor into this somehow because I've got much more experience with Palette and in that case if I understand correctly it will have a max value_count of 256 and the values will represent indexes within the palette. Whereas with color converter the values are literal color values rather than indexes mapping to a list, but I don't quite get the proper format for the literal colors yet.

Once I feel like I've got a better grasp on it I'll push a new commit to change this to the larger value if that does turn out to be needed for those Color Converter bitmaps.

Reply:
After some more experimentation I set the maximum value_count to `65535` which is the same as the effective maximum prior to this change it allows 16bit bitmaps with rgb565 colorspace